packages/application-shell/package.json

```diff
@@ -95,11 +95,11 @@
   },
   "devDependencies": {
     "@commercetools-frontend/ui-kit": "10.1.2",
-    "@testing-library/react": "8.0.7",
+    "@testing-library/react": "9.1.0",
     "react": "16.8.6",
     "react-apollo": "2.5.8",
     "react-dom": "16.8.6",
-    "react-intl": "3.1.1",
+    "react-intl": "3.1.6",
```

Comment:
Praying that this won't break the build.

Reply:
Actually they removed the `mjs` part, after they got some pressure: https://github.com/formatjs/react-intl/issues/1395#issuecomment-520185742

Can you remove the workarounds we did?

Reply:
Sure. Did so in the last commit. One note: can we maybe be a bit more conservative with these things next time. It doesn't hurt too much adopting a new version 2-4 weeks late to the party and saves us time and energy :)

Reply:
I agree...I honestly didn't expect all those problems with react-intl.